### PR TITLE
Update project-26-baybes.md

### PR DESCRIPTION
**Pull request by Scienfitz**
Updating the project page to link to the associated github repo.

We've completed the content of our repo with the investigation results. We see quite a few interesting trends which we plan to study further and could even test transfer learning on real world data.

### CODE REVIEW

**Comment by sgbaird** [comment]
Awesome! Looking forward to checking it out. Can you also include the video ID for a YouTube video and a link to a social media post on LinkedIn or X?

**Comment by Scienfitz** [comment]
sure I will make a Linkedin and X post on monday and update here. Where would the link go in the project file?

**Comment by sgbaird** [comment]
@Scienfitz awesome, thanks! The LinkedIn or X post goes before the reference section, something like Check out our [submission post on LinkedIn](link)!

**Comment by marcelmuellergdi** [comment]
The duplicated PR has been closed; however, we should maybe take over the format corrections from https://github.com/AC-BO-Hackathon/ac-bo-hackathon.github.io/pull/154/commits/5a6cdc5fa14b1a8d9285ed0763846b29cae09c2d.

**Comment by Scienfitz** [comment]
@sgbaird done, we wont produce a video tho, hope thats fine

**Comment by sgbaird** [comment]
@Scienfitz thanks! Is the issue with YouTube as a platform? It doesn't have to be on YouTube specifically, but for the manuscript, a 2-minute or less video is required. This could be posted on LinkedIn, X, or somewhere else. It just needs to be publicly accessible. This doesn't preclude you or Alex from the manuscript, but it's needed for the other members to contribute. In the simplest form, this could be a short voiceover of the poster. Is this feasible?

**Comment by sgbaird** [comment]
> @sgbaird Got it. In this case, we will contribute a video on our findings that we can publish on Youtube (the platform is not an issue for us).

Thank you! 🙏 

> Would it be fine for you if we submit a voice-over video about our GitHub `README.md` ([AC-BO-Hackathon/project-26-multiple-context-bo@`main`/README.md](https://github.com/AC-BO-Hackathon/project-26-multiple-context-bo/blob/main/README.md?rgh-link-date=2024-04-09T12%3A38%3A18Z)) and all related figures with some quick explanations, until Thursday?

Yes, that sounds great.

File: _projects/project-26-baybes.md [hunk 1]
```diff
@@ -3,18 +3,18 @@ number: 26 # leave as-is, maintainers will adjust
...
 # Comment these lines by prepending the pound symbol (#) to each line to hide these elements
 contributors:
-  - Martin Fitzner (Merck KGaA / EMD group, @Scienfitz)
-  - Alexander Hopp (Merck KGaA / EMD group, @AVHopp)
+  - Martin Fitzner (Merck KGaA / EMD group) @Scienfitz
+  - Alexander Hopp (Merck KGaA / EMD group) @AVHopp
 
-# github: AC-BO-Hackathon/<your-repo-name>
+github: AC-BO-Hackathon/project-26-multiple-context-bo
 # youtube_video: <your-video-id>
```
Comment (marcelmuellergdi) [inline _projects/project-26-baybes.md:18]:
```suggestion
youtube_video: [AC BO Hackathon: Project 26 – Multiple-Context Bayesian Optimization](https://youtu.be/wK266A0TvZ4)
```

Addition of YouTube-Link for the video contribution.

**Review by marcelmuellergdi — COMMENTED** [review]
Added the link to the video summary of our contributions.

**Pull request merged**